services/QuillLMS/client/app/bundles/Diagnostic/components/eslDiagnostic/languagePage.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 
-import { ENGLISH, languages, languagesV2, languageData, languageDataV2 } from '../../modules/translation/languagePageInfo';
+import { ENGLISH, languages, languageData } from '../../modules/translation/languagePageInfo';
 import { TrackAnalyticsEvent} from '../../actions/analytics'
 import { Events } from '../../modules/analytics'
 
 export class LanguagePage extends React.Component {
 
   handleClickLanguage = (e) => {
     const language = e.currentTarget.value;
-    const { diagnosticID, dispatch, setLanguage, previewMode, begin } = this.props;
-    if(language !== ENGLISH && diagnosticID !== 'ell') {
+    const { dispatch, setLanguage, previewMode, begin } = this.props;
+    if(process.env.NODE_ENV === 'production' && language !== ENGLISH) {
```

Comment:
Not sure if we need to do this now, but it might make sense to put the `if (process.env...` in the actual `TrackAnalyticsEvent` function instead of here, because presumably we always want to track analytics only in production and it isn't specific to this instance.